.github/workflows/cherryPick.yml

```diff
@@ -27,6 +27,7 @@ jobs:
         id: setupGitForOSBotify
         uses: ./.github/actions/composite/setupGitForOSBotifyApp
         with:
+          OP_VAULT: ${{ vars.OP_VAULT }}
```

Comment:
> We can't use the vars context in actions.

But we can here?

Reply:
I need to learn about the difference between vars/input. Give me a minute

Reply:
yes. it's available in workflows but not actions